code/DepMapGeneEss/ConvertGeneEssInputData.R

```diff
@@ -0,0 +1,70 @@
+library(tidyverse)
+
+dataFolder = "C:/Work/MatlabCode/components/human-GEM/Human-GEMDepMapEval/Human-GEM/code/DepMapGeneEss/data/"
+
+
+d = read_csv(paste0(dataFolder, "CCLE_expression_full.csv"))
```

Comment:
Here is a suggestion of how to make this path relative, given that the current directory is "Human-GEM" (perhaps the setwd should be added?).
```suggestion
dataFolder = normalizePath("./code/DepMapGeneEss/data")

d = read_csv(paste0(dataFolder, "/CCLE_expression_full.csv"))
```

Reply:
So, this doesn't solve the problem I'm afraid, I think it cannot be solved. The problem with the code above is that before you run it, you need to manually do setwd('your local path'). This means that every time you open this in R studio, you need to go through this manual procedure. With the current code, at least you only need to do it once, or zero times if it is me :). I therefore think we should leave it as is.

Reply:
I see your point. At the same time, both RAVEN and Human-GEM have installation instructions that require one to add folders to a path before proceeding.
Looking more into it, I think I might have found an improvement from the previous suggestion:

```suggestion
setwd(dirname(rstudioapi::getActiveDocumentContext()$path))

d = read_csv(paste0(rstudioapi::getActiveDocumentContext(), "/data/CCLE_expression_full.csv"))
```

What do you think @johan-gson, could this work? Note, it's relying on this code being run from RStudio, but perhaps that is fair.

Reply:
This reminds me, R/RStudio should be mentioned in the Readme.